specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/examples/DeletedAccountList.json

```diff
@@ -9,28 +9,22 @@
       "body": {
         "value": [
           {
-            "id": "/subscriptions/{subscription-id}/providers/Microsoft.Storage/locations/eastus/deletedAccounts/sto1125",
+            "key": "sto1125|2020-08-17T03:35:37.4588848Z",
```

Comment:
the key here seems to be redundant. it is a combination of account name and account creation time I guess. Why do we need this?

Reply:
That's a valid point, but I think it is a separate issue. Purpose of this PR is to ensure the response on the doc complies with the actual web response.

Reply:
Shall we mark this as resolved?